pom.xml

```diff
@@ -806,32 +436,6 @@ under the License.
 							<artifactId>maven-shade-plugin</artifactId>
 							<version>3.2.4</version>
 						</plugin>
-						<plugin>
-							<groupId>com.github.siom79.japicmp</groupId>
-							<artifactId>japicmp-maven-plugin</artifactId>
```

Comment:
Did you remove the japicmp plugin intentionally?

Reply:
Yes but now restoring it :)